R/Figures.R

```diff
@@ -864,30 +864,62 @@ TADA_TwoCharacteristicScatterplot <- function(.data, id_cols = "TADA.ComparableD
   }
 
   if (!"TADA.ComparableDataIdentifier" %in% id_cols) {
```

Comment:
Should TADA.ComparableDataIdentifier be required/automatic for id_cols? It makes the most sense to use here (so that characteristic and unit will be comparable for all results). What other options do you think might be used here?